spec/controllers/districts_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```

Comment:
This file is in the wrong location. 
Should be spec/controllers/analytics/district_controller_spec.rb

Reply:
Good catch, fixing.